.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "recommendations": [
+    "streetsidesoftware.code-spell-checker",
+    "esbenp.prettier-vscode",
+    "bradlc.vscode-tailwindcss",
+    "dbaeumer.vscode-eslint",
+    "toba.vsfire",
+    "christian-kohler.npm-intellisense",
```

Comment:
I vote to remove any extensions that others may not want to install since VSCode annoys you with a prompt to install every time you open the project.

Reply:
Go ahead and remove whatever you think doesn't belong.